frontend/src/app/main/exports.cljs

```diff
@@ -77,26 +78,42 @@
                          :is-child-selected? true
                          :childs childs}]))))
 
+(defn svg-raw-wrapper-factory
+  [objects]
+  (let [shape-wrapper (shape-wrapper-factory objects)
+        svg-raw-shape   (svg-raw/svg-raw-shape shape-wrapper)]
+    (mf/fnc svg-raw-wrapper
+      [{:keys [shape frame] :as props}]
+      (let [childs (mapv #(get objects %) (:shapes shape))]
+        [:& svg-raw-shape {:frame frame
+                         :shape shape
+                         :childs childs}]))))
+
 (defn shape-wrapper-factory
   [objects]
   (mf/fnc shape-wrapper
     [{:keys [frame shape] :as props}]
     (let [group-wrapper (mf/use-memo (mf/deps objects) #(group-wrapper-factory objects))
+          svg-raw-wrapper (mf/use-memo (mf/deps objects) #(svg-raw-wrapper-factory objects))
           frame-wrapper (mf/use-memo (mf/deps objects) #(frame-wrapper-factory objects))]
       (when (and shape (not (:hidden shape)))
         (let [shape (-> (gsh/transform-shape shape)
                         (gsh/translate-to-frame frame))
               opts #js {:shape shape}]
-          [:> shape-container {:shape shape}
-           (case (:type shape)
-             :text   [:> text/text-shape opts]
-             :rect   [:> rect/rect-shape opts]
-             :path   [:> path/path-shape opts]
-             :image  [:> image/image-shape opts]
-             :circle [:> circle/circle-shape opts]
-             :frame  [:> frame-wrapper {:shape shape}]
-             :group  [:> group-wrapper {:shape shape :frame frame}]
-             nil)])))))
+          (if (and (= :svg-raw (:type shape))
+                   (not= :svg (get-in shape [:content :tag])))
```

Comment:
Why we have 2 ways to identify the raw-svg shapes?

Reply:
The issue with the SVG is that certain elements break if they are wrapped inside a group (that what shape-container create). So, only the svg-raw in the root (<svg> tag) will be wrapped. The other elements will not be wrapped inside a group